helix-term/src/commands.rs

```diff
@@ -4626,6 +4626,15 @@ fn join_selections_impl(cx: &mut Context, select_space: bool) {
     let text = doc.text();
     let slice = text.slice(..);
 
+    let comment_prefixes = doc
```

Comment:
minor naming nit: let's call these `comment_tokens` to match the language we use for configuration